src/main/java/org/jboss/windup/plugin/WindupMojo.java

```diff
@@ -192,9 +201,12 @@ public void execute() throws MojoExecutionException
         windupConfiguration.setOptionValue(EnableTattletaleReportOption.NAME, enableTattletale == Boolean.TRUE);
         windupConfiguration.setExportingCSV(exportCSV == Boolean.TRUE);
 
-        downloadAndUnzipRules();
-
-        windupConfiguration.addDefaultUserRulesDirectory(PathUtil.getWindupRulesDir());
+        // If they have specified a path to the windup home, then just use the rules from it instead of this process.
+        if (!windupHomeSpecified)
+        {
+            downloadAndUnzipRules();
+            windupConfiguration.addDefaultUserRulesDirectory(PathUtil.getWindupRulesDir());
```

Comment:
I think this line should be out of the the `if` block because if there's the `windupHome` property, then the `rules` folder from the Windup home is not added to the set of rules folders.